### PR TITLE
Fix identity issuance and recovery errors.

### PR DESCRIPTION
**Pull request by shjortConcordium**
## Purpose

Unblock identity issuance and recovery.

## Changes

- Change accountInfo checks to handle error thrown when account does not exist.
- Fixed some types.

## Checklist

- [x] My code follows the style of this project.
- [x] The code compiles without warnings.
- [x] I have performed a self-review of the changes.
- [x] I have documented my code, in particular the intent of the
      hard-to-understand areas.
- [ ] (If necessary) I have updated the CHANGELOG.

### CODE REVIEW

File: app/pages/IdentityIssuance/PickProvider/PickProvider.tsx [hunk 1]
```diff
@@ -54,14 +54,23 @@ async function hasPrfKeySeedBeenUsed(
...
+    } catch {
+        // The seed was not used with version 0;
     }
     // Check if this index have been used with version 1 keygen.
     credId = await computeCredIdFromSeed(prfKeySeed, 0, global, 1);
-    info = await getAccountInfoOfCredential(credId, blockHash);
-    return Boolean(info);
+    try {
+        info = await getAccountInfoOfCredential(credId, blockHash);
+        return Boolean(info);
```
Comment (orhoj) [inline app/pages/IdentityIssuance/PickProvider/PickProvider.tsx:70]: It doesn't make sense to do `Boolean(info)` when `info` can't be `undefined`.
```suggestion
        return true;
```

File: app/components/PickBakerStakeAmount.tsx [hunk 2]
```diff
@@ -45,7 +45,7 @@ function useCapitalBoundCheck(
     accountInfo: AccountInfo | undefined,
     stake: string
 ): { showWarning: true; limitAfterUpdate: bigint } | { showWarning: false } {
-    const rewardStatus = useAsyncMemo(getRewardStatusLatest);
+    const rewardStatus = useAsyncMemo(getRewardStatusLatest, noOp, []);
```
Comment (orhoj) [inline app/components/PickBakerStakeAmount.tsx:48]: What does this have to do with the fix?

**Review by orhoj — CHANGES_REQUESTED** [review]

File: app/components/PickBakerStakeAmount.tsx (hunk 2 above)
Comment (shjortConcordium) [reply app/components/PickBakerStakeAmount.tsx:48]: I saw this and changed it to not run every time (because the dep array was undefined)

**Comment by shjortConcordium** [comment]
There is still issues with the identity provider, but it seems that it is a identity-provider-service/wallet-proxy issue. :thinking: 
(Maybe the fix is to migrate to the v1 issuance)

**Comment by orhoj** [comment]
> There is still issues with the identity provider, but it seems that it is a identity-provider-service/wallet-proxy issue. 🤔 (Maybe the fix is to migrate to the v1 issuance)

It is not really in scope to migrate the issuance as well. The old issuance should still work.

**Comment by shjortConcordium** [comment]
> > There is still issues with the identity provider, but it seems that it is a identity-provider-service/wallet-proxy issue. 🤔 (Maybe the fix is to migrate to the v1 issuance)
> 
> It is not really in scope to migrate the issuance as well. The old issuance should still work.

Yes, it was an issue with the identity-provider-service. The fix are on stagenet.

**Review by orhoj — APPROVED** [review]
